### ISSUE TITLE
Time machine: Widgets are not respecting the publish//expire dates

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#18795]
We have problems trying to publish//expire contents in content types that have widgets. 

If you add a publish//expire fields to the (blog) content type and then add a new piece of content, but set that should be published in some specific date in the future and expire some days after. When you go to the future time machine snapshot, we are not respecting those dates 

##To recreate 
- Login as admin 
- Go to the blog content type and add a date-time field and call this expire
- In the content type properties configure those fields to publish and expire automatically 
- Add a new piece of content to blog 
- Go to the future time machine snapshot and navigate to Blogs
- Make sure we are respecting those dates 

Actually we are showing the added piece of content in dates out of the selected interval 

##Testing info
Tested on release-5.3.3 // Postgres // FF

**Comment by freddyucv** [comment]
this is a starter bug, you have to change the file '/application/vtl/blog/blog-listing.vtl':

when it did
```

#if($request.getParameter("tag"))
    #set($tag = $request.getParameter("tag"))
    #set($hasTag = "true")
    #set($blogs = $dotcontent.pullPerPage("+contentType:Blog +Blog.tags:\"$tag\"", $page, 6,"Blog.postingDate desc"))
#else
    #set($blogs = $dotcontent.pullPerPage("+contentType:Blog", $page, 6, "Blog.postingDate desc"))
#end
```

you have to do:
```

#if($request.getParameter("tag"))
    #set($tag = $request.getParameter("tag"))
    #set($hasTag = "true")
    #set($blogs = $dotcontent.pullPerPage("+contentType:Blog +Blog.tags:\"$tag\" **+live:true**", $page, 6,"Blog.postingDate desc"))
#else
    #set($blogs = $dotcontent.pullPerPage("+contentType:Blog **+live:true**", $page, 6, "Blog.postingDate desc"))
#end
```

**Comment by freddyucv** [comment]
PR:https://github.com/dotCMS/core/pull/18863

**Comment by DeanGonzalez** [comment]
I tested branch checkout and the future and past time machine is now working and the time machine respects widgets and content that is published/unpublished in the dynamic widgets - great fix!

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.6 // Postgres // FF
